app/views/homepage/index.html.erb

```diff
@@ -17,8 +17,10 @@
 </body>
 <div class = "col-sm-6">
 <h1>Stats</h1>
-<h3>Total Outstanding Money Owed: £<%= @totalmoney %></h3>
-
+<h3>Unpaid, incomplete bookings: <%= number_to_currency(@unpaidincomplete, :unit => "£") %></h3>
```

Comment:
`number_to_currency(@unpaidincomplete, :unit => "£")`
is the same as
`number_to_currency(@unpaidincomplete, unit: '£')`

Typically we would use the second syntax with ruby (although it makes very little difference) because:
* the `key: value` syntax is more concise so easier to read than `key => value`
* `'` and `"` have slightly different meanings - `"` may contain a value for interpolation, whereas `'` may not. That means that the interpreter doesn't need to scan the string value to look for characters to replace or escape.

In reality it makes very little difference to the speed, but stylistically it is better to be consistent (rubocop would make you use `'` if this were a ruby file)